Readme.md

```diff
@@ -4,8 +4,11 @@ This repo will implement the Psiphon OONI test.
 
 ## Build instructions
 
-Ensure you have in the root of this directory the a shared library build of
-psiphon called `psi.dylib`.
+Ensure you have in the root of this directory a shared library build of
+psiphon called `libpsiphontunnel.dylib` and under `vendor/` the corresponding
+`libpsiphontunnel.h` file. Official releases can be found at
+https://github.com/Psiphon-Labs/psiphon-tunnel-core/releases in
+`Psiphon-Client-Library.zip`.
```

Comment:
I guess to support building for multiple different targets we will have to change how these paths are managed.

I see that the `Psiphon-Client-Library.zip` already includes builds for every possible target architecture and platform, so it's probably just a matter of coming up with some cmake magic to handle that.

Listing:

```
Psiphon-Client-Library
├── android
│   ├── arm64
│   │   ├── libpsiphontunnel.h
│   │   └── libpsiphontunnel.so
│   └── arm7
│       ├── libpsiphontunnel.h
│       └── libpsiphontunnel.so
├── darwin
│   ├── ios
│   │   ├── arm64
│   │   │   ├── libpsiphontunnel.a
│   │   │   └── libpsiphontunnel.h
│   │   └── arm7
│   │       ├── libpsiphontunnel.a
│   │       └── libpsiphontunnel.h
│   └── macos
│       ├── 386
│       │   ├── libpsiphontunnel.dylib
│       │   └── libpsiphontunnel.h
│       └── amd64
│           ├── libpsiphontunnel.dylib
│           └── libpsiphontunnel.h
├── linux
│   ├── 386
│   │   ├── libpsiphontunnel.h
│   │   └── libpsiphontunnel.so
│   └── amd64
│       ├── libpsiphontunnel.h
│       └── libpsiphontunnel.so
└── windows
    ├── 386
    │   ├── psiphontunnel.dll
    │   └── psiphontunnel.h
    └── amd64
        ├── psiphontunnel.dll
        └── psiphontunnel.h
```